frontend/src/metabase/parameters/components/ParameterSettings/ParameterSettings.tsx

```diff
@@ -83,6 +103,27 @@ export const ParameterSettings = ({
   const isEmbeddedDisabled = embeddedParameterVisibility === "disabled";
   const isMultiValue = getIsMultiSelect(parameter) ? "multi" : "single";
 
+  const filterOperatorData = useMemo(() => {
+    if (!sectionId) {
+      return [];
+    }
+
+    const currentSection = parameterSections.find(
+      section => section.id === sectionId,
+    );
+
+    if (!currentSection) {
+      return [];
+    }
+
+    const options = currentSection.options as SectionOption[];
+
+    return options.map(option => ({
+      label: option.sidebarMenuName ?? option.name,
```

Comment:
Can you please explain this cast?

I couldn't find `sidebarMenuName` anywhere in the project.

```suggestion
    return currentSection.options.map(option => ({
      label: option.name,
```

Reply:
I had a bigger change and just forgot to remove this line, nice catch!